app/src/main/kotlin/cz/covid19cz/erouska/service/CovidService.kt

```diff
@@ -120,6 +121,7 @@ class CovidService : Service() {
     override fun onCreate() {
         super.onCreate()
         subscribeToReceivers()
+        updateAppShortcuts()
```

Comment:
One thing I just realized is that you should update the shortcut from onDestroy. 
I'm wondering what;s gonna happen when the app is force closed. We can also tweak this later

Reply:
I can try, but deadline from @davidvavra has just been crossed :smile:

Reply:
Lol, :shipit:

Reply:
Unfortunately, the shortcut is not refreshed even when I try to do it in `onDestroy`. However, this is so big corner-case, I would just ignore it. One must count with _werid behaviors when killing some app, right?